content/community/organization/governance.html

```diff
@@ -8,13 +8,39 @@
 The following organisations are the driving force behind the open source development of Haiku:
 
 <HR>
+
+<h2>The Haiku development team</h2>
+
+<p>A group of people known as the <em>development team</em> runs and manages the project.
+They have the permissions to edit and triage tickets on the bugtracerk, push and merge
+code to the repositories, and also get voting rights in important project decisions.</p>
+
+<p>The development team is not a legal entity, but a group of people managing the technical
+aspects of the project: coding guidelines, tools to use for collaboration, and general
+direction of the development of Haiku.</p>
+
+<p>People are invited to become part of the development team after they have contributed
+sufficient work (for example in the form of patches, bug reports submissios, ...)
+and show a good understanding of the "Haiku Way" (the overall direction of the project
```

Comment:
Not sure if the first person is the right way approach this.

```suggestion
 and appear to understand the "Haiku way" (which is a term we use to describe our philosophy and the overall direction of the project
```